### PR TITLE
Add script to allow easily turning xdebug off.

### PR DESCRIPTION
**Pull request by oknate**
**Description:**
I noticed the docker is very slow, and that xdebug is enabled by default in the image.  It ignores the XDEBUG_ENABLE environment variable.  Xdebug slows down PHP, but it is very useful.  Based on a concept from the ddev command `ddev xdebug on` and `ddev xdebug off`, I added a script and ahoy command to turn xdebug on and off.

Moche recently helped me fix this underlying issue, so now  XDEBUG_ENABLE is disabled by default.  But even so, this provides a useful tool when Xdebug is enabled.  By commenting out the plugin and restarting apache, you can turn it on and off quickly.

**To Test:**
- [ ] run docker-compose up
- [ ] run `ahoy xdebug off`
- [ ] notice page speeds are faster and running `php -i | grep xdebug` from within the drupal docker container shows xdebug is not enabled within PHP.
- [ ] run `ahoy xdebug on`
- [ ] notice that xdebug is enabled.

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

**Comment by weekbeforenext** [comment]
Works for me. I like the ability to toggle xdebug. I approve :)

File: .env.example [hunk 1]
```diff
@@ -33,7 +33,7 @@ MASS_CDN_TOKEN=
 
 # XDebug defaults to Off in the drupal container (IDEKEY=PHPSTORM, Port 9000).
 #
-# Uncomment to enable XDebug.
+# Uncomment to enable XDebug (Currently not working, always enabled in image).
```
Comment (weitzman) [inline .env.example:36]: lets remove this diff

File: README.md [hunk 2]
```diff
@@ -28,6 +28,13 @@ See the [Table of Contents](/docs/README.md) for additional documentation relate
     1. **Mac/Linux:** `/etc/hosts`
     1. **Windows:** `c:\windows\system32\drivers\etc\hosts`
 
+1. Run `docker-compose up` from project root.
+
+1. Xdebug can slow down local environment. If not actively using it, you can
+   disable it.
+   - To disable: `ahoy xdebug off`
+   - To enable: `ahoy xdebug on`
+
```
Comment (weitzman) [inline README.md:37]: lets remove this diff

File: composer.json [hunk 3]
```diff
@@ -115,6 +115,7 @@
         "drupal/config_log": "^2",
         "drupal/core-composer-scaffold": "~8.9.8",
         "drupal/core-recommended": "~8.9.8",
+        "drupal/csv_field": "^1.0@alpha",
```
Comment (weitzman) [inline composer.json:118]: unrelated

File: composer.lock [hunk 4]
```diff
@@ -4114,6 +4114,53 @@
                 "issues": "https://www.drupal.org/project/issues/crop"
             }
         },
+        {
+            "name": "drupal/csv_field",
```
Comment (weitzman) [inline composer.lock:4118]: unrelated

File: .ahoy.yml [hunk 5]
```diff
@@ -67,3 +67,7 @@ commands:
   nightcrawler:
     usage: Run NightcrawlerJS fatal error tests.
     cmd: ahoy -f ${MASS_DEV_ENV:=docker}.ahoy.yml nightcrawler $@
+
+  xdebug:
+    usage: Turn xdebug "on" or "off"
```
Comment (weitzman) [inline .ahoy.yml:72]:
```suggestion
    usage: Pass "on" or "off" to enable or disable Xdebug
```

**Review by weitzman — CHANGES_REQUESTED** [review]

**Comment by oknate** [comment]
I'm going to open a new PR that's not from a fork.

**Pull request closed without merging**